PushNotifications.md

```diff
@@ -253,19 +253,34 @@ To create local notifications with more content or user interaction options once
     - To do this natively on Android instead, you could use [`notificationManager.notify`](https://developer.android.com/reference/android/app/NotificationManager#notify(int,%20android.app.Notification)). 
     - To do this natively on iOS, you can send a background message and follow [Scheduling a Notification Locally from Your App](https://developer.apple.com/documentation/usernotifications/scheduling_a_notification_locally_from_your_app). However, on iOS, you could also [customize the appearance of an alert notification](https://developer.apple.com/documentation/usernotificationsui/customizing_the_appearance_of_notifications), by registering and implementing a [`UNNotificationContentExtension`](https://developer.apple.com/documentation/usernotificationsui/unnotificationcontentextension).
 
-#### Foreground notifications
+The SDK does this by implementing [`userNotificationCenter(_:willPresent:withCompletionHandler:)`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter). This feature is not available on Android.
 
-On iOS, ably-flutter allows you to configure notification sent directly from Ably to be shown when the app is in foreground. To configure if a notification is shown in the foreground:
+#### Data Message / Background Notification
+
+- On iOS, if a notification is also present, it will be shown before `onMessage` or the callback passed to `ably.Push.notificationEvents.setOnBackgroundMessage` is called.
```

Comment:
I think this would also be nice if you separate it to two sub bullet points. Something like 
 It will be shown before
* onMessage is called 
or
* ably... called

Reply:
Done in 9063782c61f870c900baa549766f2dc27fbc4853, it's clearer now

Reply:
A couple of minor points regarding @ben-xD's [resolution](https://github.com/ably/ably-flutter/pull/227#discussion_r753020104) to @ikbalkaya's opening comment here:

1. GitHub used to require four spaces to indent lists - I think they've clearly fixed that more recently to support two - I find this curious, but I suspect we probably have conformed to four spaces elsewhere
2. The sentences ending with "it will be shown" should ideally have a colon at the end

Neither are critical, however, so I'll resolve this conversation on @ikbalkaya's behalf.